include/birch_pc.h

```diff
@@ -1,6 +1,6 @@
 #ifndef GUARD_BIRCH_PC_H
 #define GUARD_BIRCH_PC_H
 
-const u8 *GetPokedexRatingText(u16 count);
+const u8 *GetPokedexRatingText(u32 count);
```

Comment:
afaik, there's no point of changing this to `u32` if you're not changing the variable where this function is called.
In this case, only `gSpecialVar_0x8004` is used, so changing this data type is out of scope.

Reply:
Nvm, I was wrong.